extensions/wikia/PortableInfoboxBuilder/js/PortableInfoboxBuilderPonto.js

```diff
@@ -19,7 +19,7 @@ define('wikia.infoboxBuilder.ponto', ['wikia.window', 'ponto'], function (window
 		 * @returns {Boolean}
 		 */
 		this.redirectToTemplatePage = function (title) {
-			window.location = window.location.origin + '/wiki/Template:' + title;
+			w.location = w.location.origin + '/wiki/Template:' + title;
```

Comment:
It's not, and actually we're purposefully supporting only NS 10. However, https://github.com/Wikia/app/pull/9865/files#diff-d5bde9afc6a40ff66cf72bf44a3fc4b2R234 still holds true in regard to the NS name.